classes/pdf/PDF.php

```diff
@@ -51,13 +51,14 @@ public function __construct($objects, $template, $smarty, $orientation = 'P')
     {
         $this->pdf_renderer = new PDFGenerator((bool)Configuration::get('PS_PDF_USE_CACHE'), $orientation);
         $this->template = $template;
+        $smarty->escape_html = false;
```

Comment:
Is `$smarty` the global smarty object here? If so, I'm afraid you're actually disabling the autoescape for everything and not just the PDF!